src/core/CCVector.ml

```diff
@@ -243,12 +243,22 @@ let append_list a b = match b with
 *)
 
 let equal eq v1 v2 =
-  let n = min v1.size v2.size in
+  v1.size == v2.size
```

Comment:
I guess it doesn't matter in this case, but it does bother me since it's implementation dependent. Could it have issues under js_of_ocaml?